raiden_installer/web.py

```diff
@@ -370,42 +371,10 @@ def _run_swap(self, **kw):
                 if total_service_token_balance < required.service_token:
                     raise ExchangeError("Exchange was not successful")
 
-                elif service_token_balance.as_wei > 0:
+                elif token_ticker == service_token.ticker and service_token_balance.as_wei > 0:
```

Comment:
why does this not check for 
```suggestion
                elif token_ticker == service_token.ticker and service_token_balance.as_wei > required.service_token:
```
?

Reply:
One small change  `elif token_ticker == service_token.ticker and service_token_balance > required.service_token:`.(no as_wei) I will make the change.